app/src/main/java/ch/epfl/polychef/Miniatures.java

```diff
@@ -2,4 +2,12 @@
 
 public interface Miniatures {
 
+    String getKey();
+
+    void setKey(String key);
+
+    boolean isUser();
```

Comment:
Est ce qu'il serait possible de faire une version plus simple avec des default methode avec du code du genre : return this instanceof User;
plutôt que de les implémenter dans les sous classes ?